src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -795,12 +804,17 @@ IS 'primary key of judge';
 CREATE TABLE media
 (med_id                      INTEGER                       NOT NULL
 ,ctr_id                      INTEGER
+,met_id                      INTEGER                       NOT NULL

```

Comment:
Will you be adding the INSERTS into the standing data script we have? So that we know what should be in this table

Reply:
As I understand it, this is being added to accommodate future enhancement, and currently this value will have 1 distinct value, corresponding to AUDIO. 
If there is a realistic chance that media_type might be extended to include VIDEO, we would need a second row, and at that point, i suspect the upper limit of INTEGER might become an issue for file_size